### DISCUSSION TITLE
【Help me】I currently have two questions about GitHub Sponsor

### DISCUSSION THREAD

**Discussion by maomincoding** [community/community#144897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I currently have two questions.

First, the organization that I originally deleted and had no sponsors is still displayed in the sponsor list.
I accidentally created a Stripe Express account associated with GitHub Sponsor, which now seriously affects my use of Github Sponsor. I hope everyone can help me delete this account and I will set it up again.
These are the two questions I raised and summarized in the community. All the above conversations are centered around these two questions.

**Comment by maomincoding** [comment]
![image](https://github.com/user-attachments/assets/dabb9957-d797-4363-ac9d-9167fd32bcf6)

**Reply by maomincoding** [reply]
First, the organization that I originally deleted and had no sponsors is still displayed in the sponsor list.

**Comment by SameerNadaf** [comment]
If you have deleted a GitHub organization, but it still appears in the sponsor list, it could be due to the following reasons:

### Possible Causes
1. **Pending Cache or Data Sync:**
   - GitHub might still display the organization due to cached data. Sometimes, updates take a while to reflect across all GitHub services.

2. **Active Sponsorships:**
   - If the deleted organization had active sponsorships at the time of deletion, it might remain in the sponsor list until the sponsorships are fully processed or resolved.

3. **Associated Records:**
   - There could be historical or associated records that GitHub retains for transparency or auditing purposes, which might cause the organization to still appear in some views.

4. **System Error:**
   - A bug or issue on GitHub’s side might cause deleted organizations to persist in certain lists.

---

### Steps to Resolve
1. **Refresh and Recheck:**
   - Wait for a while (e.g., 24 hours) and clear your browser cache. Check if the organization still appears.

2. **Check Sponsorships:**
   - Visit the sponsorship settings to ensure no active or pending sponsorships are associated with the deleted organization.

3. **Contact GitHub Support:**
   - If the issue persists, reach out to GitHub Support for clarification and assistance. Provide the organization name and explain the situation.

   - Use this link to open a support ticket: [GitHub Support](https://support.github.com/contact)

4. **Double-Check Deletion:**
   - Verify that the organization was indeed deleted and not simply deactivated or transferred.

---

**Reply by SameerNadaf** [reply]
It sounds like you've done everything correctly on your end and raising a support ticket (work order) with GitHub is the best course of action now. Since the organization had no sponsorships or repositories, the issue likely stems from a backend or caching problem on GitHub's side. Here’s what you can do while you wait for their response:

---

### **Steps While Waiting for GitHub Support**

1. **Review Your Work Order Submission**
   - Ensure the ticket includes all relevant details:
     - The name of the deleted organization.
     - Confirmation that it had no repositories or sponsorships.
     - The specific URL (`https://github.com/sponsors/accounts`) where it still appears.
     - Any screenshots to illustrate the issue.

2. **Double-Check Associated Accounts**
   - Verify that the deleted organization isn’t linked to any other GitHub accounts you control. Sometimes, linked accounts or teams may retain records.

3. **Clear Local Cache**
   - While unlikely to resolve the issue, clearing your browser cache or trying a different browser ensures that the issue isn’t client-side.

4. **Follow Up on the Work Order**
   - GitHub typically responds within a few days, but if you don’t hear back within 5–7 business days, follow up with a polite reminder on the ticket.

---

### **What to Expect from GitHub Support**
GitHub Support may:
- Remove the organization from the sponsor list manually.
- Provide insight into why the organization still appears (e.g., caching, database issue, or a rare policy around deleted entities).
- Request additional information if they need more context.

---

### **Temporary Workaround**
If this issue impacts your ability to manage sponsorships or account visibility:
- Consider adding a note in your GitHub profile or README to explain the situation until GitHub resolves it.

---

**Reply by maomincoding** [reply]
Thank you for your reply. The response speed for the ticket submitted two days ago was quite slow. I have synchronized all the necessary information.

**Comment by mecodeatlas** [comment]
Hi @maomincoding 👋🏾,It looks like @SameerNadaf was able to get this solved for you. If so, could you please mark his response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Reply by maomincoding** [reply]
> Hi @maomincoding 👋🏾,It looks like @SameerNadaf was able to get this solved for you. If so, could you please mark his response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer. ![image](https://private-user-images.githubusercontent.com/14021469/353319770-b89e7161-c2d1-47e2-a058-242df633e6b2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI5MjczNjQsIm5iZiI6MTczMjkyNzA2NCwicGF0aCI6Ii8xNDAyMTQ2OS8zNTMzMTk3NzAtYjg5ZTcxNjEtYzJkMS00N2UyLWEwNTgtMjQyZGY2MzNlNmIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTMwVDAwMzc0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQzYWQ1Mjc3ODZhZmMzYTI5YjEzZGFmYmI5ZjM5N2FhOGQ1NzIyY2EzNWQ1NTI5OTk4NDQxZTNjN2ZhODczNjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WRIuOr7oJku3eE4De-JwMvv4759ESoI8WteZF-PUi5k)

I'm very sorry, but his method didn't really solve my problem.
I currently have two questions.
1. One is that an organization that I originally deleted and had no sponsorship in it is still displayed in the sponsor list.
2. I accidentally created a Stripe Express account associated with GitHub Sponsor, and now it seriously affects my use of Github Sponsor. I hope you can help me delete this account. I will set it up again.

These are the two questions I raised and summarized in the community. All the above conversations revolve around these two questions.

**Comment by maomincoding** [comment]
I currently have two questions.

1. One is that an organization that I originally deleted and had no sponsorship in it is still displayed in the sponsor list.
2. I accidentally created a Stripe Express account associated with GitHub Sponsor, and now it seriously affects my use of Github Sponsor. I hope you can help me delete this account. I will set it up again.
These are the two questions I raised and summarized in the community. All the above conversations revolve around these two questions.

**Reply by maomincoding** [reply]
> I currently have two questions.
> 
> 1. One is that an organization that I originally deleted and had no sponsorship in it is still displayed in the sponsor list.
> 2. I accidentally created a Stripe Express account associated with GitHub Sponsor, and now it seriously affects my use of Github Sponsor. I hope you can help me delete this account. I will set it up again.
>    These are the two questions I raised and summarized in the community. All the above conversations revolve around these two questions.

Regarding your second question, Stripr Express accounts can only be deleted by the platform through the API. [https://support.stripe.com/express/questions/how-can-i-delete-my-express-account](https://support.stripe.com/express/questions/how-can-i-delete-my-express-account)

So please help me delete my account after seeing this, thank you.

**Reply by maomincoding** [reply]
<img width="1099" alt="截屏2024-12-01 23 51 30" src="https://github.com/user-attachments/assets/2e96df6d-1209-40a0-a9bf-67a2fca3674e">
This is Stripe's official response.

**Comment by maomincoding** [comment]
These two questions have been bothering me for two weeks, and there is currently no effective response.

**Comment by maomincoding** [comment]
I tried to delete my Stripe account myself, but got this error.

https://github.com/sponsors/strveJS/dashboard/settings
![image](https://github.com/user-attachments/assets/031da40a-2e1b-4376-995c-9beadfae3841)
